pkg/cmd/node/node_diagnostics.go

```diff
@@ -14,7 +14,8 @@ func newCmdNodeDiagnostics(ctx api.Context) *cobra.Command {
 		Use:   "diagnostics",
 		Short: "Use diagnostics bundles",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := ctx.Deprecated("This command is deprecated since DC/OS 2.0, please use 'dcos diagnostics' instead.")
+			err := ctx.Deprecated("'dcos node diagnostics' command is deprecated since 2.0, but still functional in this release." +
```

Comment:
just a really minor nitpick: i'd write `'dcos node diagnostics' is deprecated`.

Reply:
I'd stay with this just like described in ticket.